OpenData/secular printed material - 1921 - 1990/Coraa ny Gael/Coraa ny Gael 1953-08-25/document.csv

```diff
@@ -1,31 +1,31 @@
+T’eh er-chee shassoo son ny Fir-lhee Americanagh ec chaglym vooar dy Fir-lhee ayns Lunnin.,He is about to represent the American Doctors at a big meeting of doctors in London.,
 "Keayrt dy row va eh-hene yn Reiltagh jeh’n Cheshaght dy Vanninee ayns America Twoaieagh. Laa ennagh er y chiaghtyn shoh chaie, hug eh shilley er Purt ny h-inshey, y voayl haink eh voish hoshiaght, as v’eh er ny oltaghey liorish yn ardhoieder ny varrantee ny valley. Mnr. Jamys H. Kelly.","At a certain time he himself was President of the North American Manx Association. Some day last week he went to see Peel, the place he first came from, and he was welcomed by the chairman of the town commissioners, Mr. James H. Kelly.",
-"     Tra hug Fer Lhee Moore booise daue er son y gein-vie dy dooar eh aynshen, dooyrt eh “Ta’n jishag aym cliaghtit dy rouail ooilley mygeayrt America, as yinnagh eh gra nagh row boayll erbee feeu son veg, mannagh row Manninagh beaghey aynshen, son va dy-chooilley lught-thie Vanninagh jannoo feer vooar jeh dagh Vanninagh elley yinnagh goll dy chur shilley orroo.’","     When Doctor Moore gave thanks to them for the good cheer he received there, he said “My dad is used to wandering all around America, and he would say there wasn’t any spot worth anything, unless there was a Manx person living there, because every Manx family were making a great deal of every other Manx person who would go to visit them.’",
-     Ta shin clashtyn dy vel Harry Johnson yn canteyder cheet-er dy mie lurg yn chingys vooar v’echey dy gerrid. V’eh giarit lesh skynn y Fer-lhee ayns y thie-lheeys.,     We hear that Harry Johnson the auctionerr is coming on well after the great illness he had recently. He was cut by the knife of the doctor (operated on) in the hospital.,
+"     Tra hug Fer Lhee Moore booise daue er son y gein-vie dy dooar eh aynshen, dooyrt eh “Ta’n jishag aym cliaghtit dy rouail ooilley mygeayrt America, as yinnagh eh gra nagh row boayll erbee feeu son veg, mannagh row Manninagh beaghey aynshen, son va dy-chooilley lught-thie Vanninagh jannoo feer vooar jeh dagh Vanninagh elley yinnagh goll dy chur shilley orroo.’","     When Doctor Moore gave thanks to them for the good cheer he received there, he said ‘My dad is used to wandering all around America, and he would say there wasn’t any spot worth anything, unless there was a Manx person living there, because every Manx family were making a great deal of every other Manx person who would go to visit them.’",
```

Comment:
```suggestion
"     Tra hug Fer Lhee Moore booise daue er son y gein-vie dy dooar eh aynshen, dooyrt eh “Ta’n jishag aym cliaghtit dy rouail ooilley mygeayrt America, as yinnagh eh gra nagh row boayll erbee feeu son veg, mannagh row Manninagh beaghey aynshen, son va dy-chooilley lught-thie Vanninagh jannoo feer vooar jeh dagh Vanninagh elley yinnagh goll dy chur shilley orroo."","     When Doctor Moore gave thanks to them for the good cheer he received there, he said "My dad is used to wandering all around America, and he would say there wasn’t any spot worth anything, unless there was a Manx person living there, because every Manx family were making a great deal of every other Manx person who would go to visit them."",
```

Reply:
Changed both sides to have matching double quotes

Reply:
You can't use double-quotes unless they're smart quotes. This is what the CSV uses as delimiters